### PR TITLE
[DOC] Edit supportbundle content initial

### PR DESCRIPTION
**Pull request by KoryKessel-Mirantis**
Initial edit of MKE 4 supportbundle content, including text changes and content restructuring/granulation.

### CODE REVIEW

**Comment by ry4nz** [comment]
To improve discoverability, could you consider linking this page to https://github.com/Mirantis/mke-docs/tree/main/content/releases

File: content/docs/reference/supportbundle/support-bundle.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+     - Capture of cluster information
+     - Setting of cluster resources
+     - Collection of logs from the `blueprint-controller-manager` and
+       `blueprint-operator-webhook` pods, in the `logs/` directory of the
+       output.
+
+## To collect host information using the k0s-provided YAML file
+
+1. Obtain the [k0s-provided YAML file](https://docs.k0sproject.io/stable/support-bundle-worker.yaml).
```
Comment (RaunakShah) [inline content/docs/reference/supportbundle/support-bundle.md:48]: Do we want to again call out here that k0s does not support using this tool for v1.29? Official support was only added for 1.30

Comment (KoryKessel-Mirantis) [reply content/docs/reference/supportbundle/support-bundle.md:48]: Considering the existing note isat the top of the topic and the topic is not very long, it seems like overkill to me to mention it again here.

File: content/docs/reference/supportbundle/support-bundle.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Support bundle
+
+>For the Alpha.1 release, MKE 4 deploys k0s v1.29.3, which does not inherently
+support the [Replicated support bundle
+tool](https://troubleshoot.sh/docs/support-bundle/introduction/).
+
+## To manually collect a support bundle:
```
Comment (MagdaDziadosz) [inline content/docs/reference/supportbundle/support-bundle.md:7]:
```suggestion
## Manually collect a support bundle
```

It's a header, it should not have a colon at the end

File: content/docs/reference/supportbundle/support-bundle.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+             name: logs/blueprint-system
+     ```
+
+     This configuration accomplishes the following:
+
+     - Capture of cluster information
+     - Setting of cluster resources
+     - Collection of logs from the `blueprint-controller-manager` and
+       `blueprint-operator-webhook` pods, in the `logs/` directory of the
+       output.
```
Comment (MagdaDziadosz) [inline content/docs/reference/supportbundle/support-bundle.md:44]:
```suggestion
     - Captures cluster information
     - Sets cluster resources
     - Collects logs from the `blueprint-controller-manager` and
       `blueprint-operator-webhook` pods, in the `logs/` directory of the
       output.
```

File: content/docs/reference/supportbundle/support-bundle.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+## To collect host information using the k0s-provided YAML file
```
Comment (MagdaDziadosz) [inline content/docs/reference/supportbundle/support-bundle.md:46]:
```suggestion
## Collect host information using the k0s-provided YAML file
```

File: content/docs/reference/supportbundle/support-bundle.md (hunk 2 above)
Comment (KoryKessel-Mirantis) [reply content/docs/reference/supportbundle/support-bundle.md]: My intent was for it to present like the pseudo-headers (my term) we use in the other docs sets.

File: content/docs/reference/supportbundle/support-bundle.md (hunk 4 above)
Comment (KoryKessel-Mirantis) [reply content/docs/reference/supportbundle/support-bundle.md]: My intent was for it to present like the pseudo-headers (my term) we use in the other docs sets.

**Review by MagdaDziadosz — APPROVED** [review]

**Review by RaunakShah — APPROVED** [review]

**Pull request merged**